src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
-
+import { Movie } from '../../react-app-env';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={movie.imgUrl}
+          alt={movie.title}
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{movie.title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {movie.description || null}
```

Comment:
```suggestion
        {movie.description}
```
As the task description says:

> All the fields in the movie except the `description` are required

Make this field not required in your `Movie` interface